source/features/useful-not-found-page.tsx

```diff
@@ -108,6 +109,15 @@ function init(): false | void {
 	}
 }
 
+async function initPRCommit(): Promise<void | false> {
```

Comment:
It never returns `false`

```suggestion
async function initPRCommit(): Promise<void> {
```